logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/Conversation.kt

```diff
@@ -18,6 +18,7 @@ data class Conversation(
     val mutedStatus: MutedConversationStatus,
     val lastNotificationDate: String?,
     val lastModifiedDate: String?,
+    val lastSeenDate : String?,
```

Comment:
I would prefer to call this `lastReadDate`.

Reply:
I agree as we will observe unread messages it will be more logical to use lastReadDate